Neos.Flow/composer.json

```diff
@@ -28,7 +28,7 @@
 
         "psr/http-message": "~1.0.0",
 
-        "ramsey/uuid": "^3.0.0",
+        "ramsey/uuid": "~3.7.1",
```

Comment:
Is there a good reason to constrain this so thightly onto 3.7 all of a sudden?

Reply:
Good point. Mmm, maybe it's fine after all...